DuckDB.NET/DuckDB.NET.csproj

```diff
@@ -43,12 +45,27 @@
       <PackagePath></PackagePath>
     </None>
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
-
+  
+  <!-- Download and include the native libraries into the nuget package-->
+  <Target Name="DownloadNativeLibs" BeforeTargets="GenerateAdditionalSources">
```

Comment:
Can you make this target execute only in `Release` mode?

Reply:
Done